src/main/java/seedu/intrack/ui/InternshipCard.java

```diff
@@ -65,7 +65,7 @@ public InternshipCard(Internship internship, int displayedIndex) {
         PseudoClass offered = PseudoClass.getPseudoClass("offered");
         lab.pseudoClassStateChanged(offered, (internship.getStatus().toString()).equals("Offered"));
         status.getChildren().add(lab);
-        salary.setText("$" + internship.getSalary().value);
+        upcomingTask.setText("Upcoming Task: " + internship.getTasks().get(0).toString());
```

Comment:
Will this work properly if the internship has no tasks?

Reply:
Thank you for pointing that out, I have created a guard case to anticipate a ```List<Task>``` of size 0.